utils/file/file.go

```diff
@@ -252,20 +252,23 @@ func DumpPluginModifiedComposeFiles(ctx context.Context, plugin, funcName string
 	}
 }
 
-func OverwriteFile(file string, data []byte) {
+func OverwriteFile(file string, data []byte) error {
```

Comment:
Now that `OverwriteFile` returns an error, all callers of this function should check for the error and handle it appropriately. Could you please look into its callers and make corresponding changes?

Reply:
yes. I verified that

Reply:
Interesting that there doesn't seem to be any caller of this function.